source/ada/lsp-ada_documents.adb

```diff
@@ -33,9 +33,10 @@ package body LSP.Ada_Documents is
    function To_LSP_String
      (Value : Wide_Wide_String) return LSP.Types.LSP_String;
 
-   function Get_Symbol_Kind
```

Comment:
Why do you rename this function? It returns LSP.Messages.SymbolKind; So Get_Symbol_Kind is good enough for me.

Reply:
I do not only rename it, I change its mode of operation: Before it operated on an Ada_Node, now it operates on a Decl, hence the renaming. Also, as you can see in this next commit:

https://github.com/AdaCore/ada_language_server/pull/22/commits/162c8f13d32e14acca7f98d6445dd3341fb404be

It's used as a basis to provide completion kind too. We can rename it Get_Decl_Symbol_Kind if you prefer, but ultimately it would be good to have all of those based on some function that operates on an abstract kind, in order to not duplicate the logic for every kind.

Reply:
I would keep these two kinds independent, so we will be able to change mapping from nodes to symbol kind without affecting mapping to completion kind.

Reply:
I disagree. 

1. I'm not going to duplicate code today because we might need that later. If we need different logic later, we'll take care of it later, it's simple enough.

2. There is no practical use case where you want a different kind for symbols and completions. Also keep in mind that all those kinds determine are icons.